build/webpack.config.js

```diff
@@ -70,6 +70,9 @@ module.exports = {
     chunkFilename: 'assets/js/[id].[chunkhash].js',
     pathinfo: false,
   },
+  node: {
```

Comment:
We could also try `node: false` and see if all dependencies cope with it.

Reply:
Seems ok so far! It builds, and clicking around locally is ok. I'll push the change.